styles/portfolio.css

```diff
@@ -16,53 +20,198 @@ h2 {
   grid-template-rows: 1fr 1fr;
   width: 100%;
   margin: 0 auto;
+  color: #fff;
 }
 
-.portfolio__image:nth-child(1) {
+.portfolio__imgContainer {
+  position: relative;
+}
+
+.portfolio__imgContainer:nth-child(1) {
   grid-column: 1 / span 9;
 }
-.portfolio__image:nth-child(2) {
+.portfolio__imgContainer:nth-child(2) {
   grid-column: 10 / span 7;
 }
-.portfolio__image:nth-child(3) {
+.portfolio__imgContainer:nth-child(3) {
   grid-column: 17 / span 9;
 }
-.portfolio__image:nth-child(4) {
+.portfolio__imgContainer:nth-child(4) {
   grid-column: 1 / span 7;
 }
-.portfolio__image:nth-child(5) {
+.portfolio__imgContainer:nth-child(5) {
   grid-column: 8 / span 18;
 }
+.portfolio__overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(
+    180deg,
+    rgba(34, 34, 34, 0.5) 0%,
+    rgba(255, 154, 61, 0.5) 100%
+  );
+  opacity: 0;
+  transition: 0.5s;
+}
+.portfolio__imgContainer h3 {
+  position: absolute;
+  top: 0%;
+  left: 5%;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 47px;
+  letter-spacing: 0.02em;
+}
+.portfolio__imgContainer p {
+  position: absolute;
+  font-size: 16px;
+  top: 20%;
+  left: 5%;
+}
+.portfolio__imgContainer p {
+  width: 90%;
+}
+.portfolio__imgContainer:last-child p {
+  width: 40%;
+}
+.portfolio__imgContainer button {
+  position: absolute;
+  top: 45%;
+  left: 5%;
```

Comment:
To co wyżej

Reply:
powyrzucane wszystkie pozycjonowania małych elementów